### ISSUE TITLE
A new wiki page - a Troubleshooting FAQ for hsg-project. Comments, suggestions?

### ISSUE THREAD

**Issue by joewiz** [HistoryAtState/hsg-project#35]
Hi all @HistoryAtState/editors,

I've started a new page on the wiki, https://github.com/HistoryAtState/hsg-project/wiki/troubleshooting. It only has a few items now, but these are the tried-and-true solutions to common problems I and other hsg-project users have. Please check it out and reference it when you have problems. And if you have any comments or suggestions for new entries, please let me know.

Cheers,
Joe

**Comment by mcdanielhn** [comment]
Great idea! Thanks! I have also found that eXide does not always keep me logged in as admin even when I select "Remember me." Often in the same session, I get kicked out, which messes with the S3 cache update. Maybe a reminder to double check login? I think a reminder is in the instructions, but it would also be good to have here.

**Comment by mcdanielhn** [comment]
I also really like the trick about "Opening this repository in GitHub," because the uncommitted changes don't always show up in GitHub, and it's nice to have a reminder. Again, I think this is in the instructions, but a good note here.

**Comment by KerryHite** [comment]
Great resource.  Thanks, Joe!

**Comment by joewiz** [comment]
@mcdanielhn I've seen this before too. My suggestion is to reload the eXide window before starting the S3 cache update. After you reload, it will show the actual login status. Sometimes it is stale, showing the user as "admin" when the login session has in fact expired. Could you give this a try and let me know if it works?

And thanks to both you and @KerryHite for this feedback!
